### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for buttons in "Web App-Show Properties" tabitem

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1406]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish to web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, tab to and select  "web app for container" button.
6. Fill the required data and click on "OK" button.
7. Navigate to and select "Show Azure Explorer" button at the toolbar .
8. In "Azure Explorer" , select the created app under web apps and use shift+F10 to open the context menu.
9. Navigate to and select "Show properties" menu item .
10. In "Properties" dialog , tab to buttons and observe the control type using inspect.

**Actual Result:**
Name property is empty for the button 

**Narrator behaviour:**
Tab
button, 
Tab
button, 
Tab
button

**NVDA behaviour:**
button
button
button

**Expected Result:**
Name property should be simple and meaningful.
Name property for the button should be "Add ,delete and Edit".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![properties-buttons-name-inspect](https://user-images.githubusercontent.com/22956130/35685376-8edfbf4c-078f-11e8-9cca-33c34bfe84eb.jpg)
![properties-buttons-name-narrator](https://user-images.githubusercontent.com/22956130/35685381-92269540-078f-11e8-8bc1-fe79f1c29180.jpg)
![properties-buttons-name-nvda](https://user-images.githubusercontent.com/22956130/35685385-94995d58-078f-11e8-8a4c-eafbb42c475b.jpg)

**Comment by jdneo** [comment]
Fixed

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1406-fixed](https://user-images.githubusercontent.com/22956130/39760979-7201caa4-52f4-11e8-93f2-d176d37554c0.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749881
